sdk/trace/batch_span_processor_test.go

```diff
@@ -189,14 +191,21 @@ func generateSpan(t *testing.T, option testOption) {
 	sc := getSpanContext()
 
 	for i := 0; i < option.genNumSpans; i++ {
-		sc.TraceID.High = uint64(i + 1)
+		binary.BigEndian.PutUint64(sc.TraceID[0:8], uint64(i+1))
 		_, span := apitrace.GlobalTracer().Start(context.Background(), option.name, apitrace.ChildOf(sc))
 		span.End()
 	}
 }
 
+func traceIDFromString(s string) core.TraceID {
```

Comment:
Let's make this public. It's duplicated several times already and is generally useful.

Reply:
Yes. I didn't know where to place it so I just duplicated it to make it work and fix the tests. Do you think this could go inside the `api/core` package?

Reply:
Sounds good to me.

Reply:
Done!